pyanaconda/modules/network/network.py

```diff
@@ -100,3 +117,35 @@ def set_current_hostname(self, hostname):
 
         self._hostname_service_proxy.SetHostname(hostname, False)
         log.debug("Current hostname is set to %s", hostname)
+
+    @property
+    def connected(self):
+        """Is the system connected to the network?"""
+        if self.nm_client:
```

Comment:
I personally think this will deserve abstraction. It would be nice to call `self.nm_available` instead.

Reply:
Added.